src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/resource/ResourceEditor.tsx

```diff
@@ -100,7 +101,9 @@ const ResourceEditor = ({
             bundleEntry: selectedEntry,
           };
 
-          const topElements = getTopLevelElements(selectedResource);
+          const topElements = filterElements(
```

Comment:
Could this be done in `getTopLevelElements` method itself?

Reply:
my idea was, what if the count of the elements that needs to excluded increases so i had written a seperate function for that but looks like it is something that is being used in multiple places so yeah it totally makes sense to add it at the root level so i have made the changes